src/pybind/mgr/dashboard/frontend/src/app/shared/datatable/table-key-value/table-key-value.component.ts

```diff
@@ -23,6 +23,7 @@ export class TableKeyValueComponent implements OnInit, OnChanges {
   columns: Array<CdTableColumn> = [];
 
   @Input() data: any;
+  @Input() autoReload: any = 5000;
```

Comment:
You should remove this code because [you do not use it](https://github.com/ceph/ceph/pull/21353/files#diff-b15ebd947ea9522e20511d4bbf56a5f1R14) yourself.